org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/JDTUtils.java

```diff
@@ -978,10 +978,55 @@ public static String getConstantValue(IField field, ITypeRoot typeRoot, IRegion
 		} else if (constantValue instanceof Character) {
 			return '\'' + constantValue.toString() + '\'';
 		} else {
-			return constantValue.toString(); // getHexConstantValue(constantValue);
+			String  value = constantValue.toString();
+			value+=";//0x";
+			value +=  getHexConstantValue(constantValue);
+			return value;

```

Comment:
This is safer and looks a bit better IMHO
```java
		StringBuilder value = new StringBuilder(constantValue.toString());
		String hexValue = getHexConstantValue(constantValue);
		if (hexValue != null) {
			value.append(" [hex: 0x").append(hexValue).append("]");
		}
		return value.toString();
```

<img width="324" alt="Screen Shot 2019-12-24 at 5 12 19 PM" src="https://user-images.githubusercontent.com/148698/71419632-a97dad80-2670-11ea-8c9b-cc32f54a77bc.png">

Reply:
on completion:
<img width="821" alt="Screen Shot 2019-12-24 at 5 33 05 PM" src="https://user-images.githubusercontent.com/148698/71420090-86a0c880-2673-11ea-8fa5-c59e93f1fe08.png">